panic: runtime error: slice bounds out of range

Hi, trying to test out kapacitor, following the tut @ https://docs.influxdata.com/kapacitor/v0.2/introduction/getting_started/
I get the following error, dont really know anything about go or i would try to debug further.
cheers!
rob
```
# kapacitor define -name load_longterm -type stream -tick cpu_alert.tick  -dbrp collectd
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
main.(*dbrps).Set(0xc858a0, 0x7fff9fab98bc, 0x8, 0x0, 0x0)
        /gopath/src/github.com/influxdata/kapacitor/cmd/kapacitor/main.go:358 +0x323
flag.(*FlagSet).parseOne(0xc82001a660, 0x9d5a01, 0x0, 0x0)
        /usr/local/go/src/flag/flag.go:881 +0xdd9
flag.(*FlagSet).Parse(0xc82001a660, 0xc82000a0c0, 0x8, 0x8, 0x0, 0x0)
        /usr/local/go/src/flag/flag.go:900 +0x6e
main.main()
        /gopath/src/github.com/influxdata/kapacitor/cmd/kapacitor/main.go:107 +0xa67

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 18 [syscall]:
os/signal.loop()
        /usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
        /usr/local/go/src/os/signal/signal_unix.go:28 +0x37
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -14,6 +14,7 @@ Improved UDFs, lots of bug fixes and improvements on the API. There was a breaki
 ### Bugfixes
 
 - [#177](https://github.com/influxdata/kapacitor/issues/177): Fix panic for show command on batch tasks.
+- [#185](https://github.com/influxdata/kapacitor/issues/185): Fix panic in define command with invalid dbrp value.
 
 ## v0.10.0 [2016-01-26]
 

--- a/cmd/kapacitor/main.go
+++ b/cmd/kapacitor/main.go
@@ -348,17 +348,20 @@ func (d *dbrps) String() string {
 func (d *dbrps) Set(value string) error {
 	dbrp := kapacitor.DBRP{}
 	if len(value) == 0 {
-		return fmt.Errorf("dbrp cannot be empty")
+		return errors.New("dbrp cannot be empty")
 	}
 	var n int
 	if value[0] == '"' {
 		dbrp.Database, n = parseQuotedStr(value)
 	} else {
 		n = strings.IndexRune(value, '.')
+		if n == -1 {
+			return errors.New("does not contain a '.', it must be in the form \"dbname\".\"rpname\" where the quotes are optional.")
+		}
 		dbrp.Database = value[:n]
 	}
 	if value[n] != '.' {
-		return fmt.Errorf("dbrp must specify retention policy, do you have a missing or extra '.'?")
+		return errors.New("dbrp must specify retention policy, do you have a missing or extra '.'?")
 	}
 	value = value[n+1:]
 	if value[0] == '"' {

